docs/src/fts.md

```diff
@@ -94,7 +113,10 @@ Stemming is useful for improving search results by reducing words to their root
 
 For example, to enable stemming for English:
 ```python
-table.create_fts_index("text", use_tantivy=True, tokenizer_name="en_stem")
+# Synchronous client
+table.create_fts_index("text", tokenizer_name="en_stem")
+# Asynchronous client
+await async_table.create_index("text", config=FTS(tokenizer_name="en_stem"))
```

Comment:
There is no `tokenizer_name` parameter in FTS. I think you instead want:

```suggestion
await async_table.create_index("text", config=FTS(language="English", stem=True, remove_stop_words=True))
```

Also, maybe we should move more of these into `python/tests/docs` so that the examples are tested? That would catch mistakes like this.